client/js/libs/handlebars/ircmessageparser/parseStyle.js

```diff
@@ -107,25 +143,19 @@ const properties = ["bold", "textColor", "bgColor", "italic", "underline", "reve
 
 function prepare(text) {
 	return parseStyle(text)
-		.filter(fragment => fragment.text.length)
```

Comment:
This traversal is unnecessary as L36 (now R52) ensures the fragment does not get created if the text length is zero.